### PR TITLE
Added tests for Python 3.10

### PR DESCRIPTION
**Pull request by GitRon**
Just saw that we are not up-to-date 🙂

### CODE REVIEW

**Comment by GitRon** [comment]
@costela fyi

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
           - 3.7
           - 3.8
           - 3.9
+          - "3.10"
```
Comment (costela) [inline .github/workflows/workflow.yaml:22]: nitpicking: can we use the same format for the whole list?

**Review by costela — CHANGES_REQUESTED** [review]
thanks! good catch!

just a small nitpick :sweat_smile:

**Comment by GitRon** [comment]
@costela done

**Review by costela — APPROVED** [review]

**Pull request merged**